app/src/main/java/org/mozilla/fenix/compose/ComposeViewHolder.kt

```diff
@@ -31,12 +32,18 @@ abstract class ComposeViewHolder(
     @Composable
     abstract fun Content()
 
+    /**
+     * Optional override used to disable private browsing theming and only obey dark/light theming.
+     */
+    @Composable
+    open fun allowPrivateBrowsingTheme(): Boolean = true
```

Comment:
```suggestion
     */
    @Composable
    open fun usePrivateTheme(): Boolean = true
```
I actually preferred your initial naming of this since it was less verbose while still conveying the exact meaning.

Reply:
oh ok, I was worried it could be misconstrued. I'll revert it

Reply:
Alternatively, we could say `allowPrivateTheme`.